src/chain.h

```diff
@@ -182,6 +186,7 @@ class CBlockIndex
         nTx = 0;
         nChainTx = 0;
         nStatus = 0;
+        nBranchId = 0;
```

Comment:
add to the ZIP that you are including the branch_id in the block

Reply:
This is the block index, not the block itself. The inclusion here is just an implementation optimisation.